omero/developers/Server/FS.txt

```diff
@@ -26,4 +26,4 @@ within the OMERO clients.
     :maxdepth: 1
     :titlesonly:
 
-    /developers/ManagedRepository/ManagedRepository
+    /developers/ImportFS
```

Comment:
I'd either have both dropbox and import in a single bulleted list, or handle it via the paragraphs.